### ISSUE TITLE
'Follow'ing updates

### ISSUE THREAD

**Issue by SiIason** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#62]
Currently if followers=True then your follows are loaded at start-up only.

A custom (run.py setting) hourly value allowing periodically checking would be helpful.
So you can make changes to followers without restarting the script, if remotely located for example.

**Comment by testdev123456** [comment]
yes, would be great to regularly update the manual 'Array of streamers' or the followers=True list.

**Comment by Tkd-Alex** [comment]
https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/pull/75

Testers ? :rofl:

**Comment by Tkd-Alex** [comment]
Someone has tested the PR?
